crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -103,7 +103,10 @@ def dump_openstack_database
       node.save
     end
 
-    raise "There does not seem to exist any node running the database." if db_nodes.empty?
+    # Check if database proposal exists in case when somebody is trying to upgrade undeployed nodes
+    # or only with Ceph cluster
+    db_proposal = Proposal.where(barclamp: "database", name: "default").first
+    raise "There does not seem to exist any node running the database." if db_proposal && db_nodes.empty?
```

Comment:
Metrics/LineLength: Line is too long. [105/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)